episodes/13-synchronization.md

```diff
@@ -177,22 +173,23 @@ this is new task working: 9
 this is new task working: 10
 New task finished
 and now it is done
-~~~
-{: .output}
+```
 
-> ## Discussion
->
-> What would happen if we assign a value to _x_ right before launching the new
-> task? What would happen if we assign a value to _x_ right before launching
-> the new task and after the _writeln("and now it is done");_ statement?
->
-> Discuss your observations.
-{: .discussion}
+::::::::::::::::::::::::::::::::::::::: discussion
+
+## Discussion
+
+What would happen if we assign a value to _x_ right before launching the new task? What would happen if we
+assign a value to _x_ right before launching the new task and after the _writeln("and now it is done");_
+statement?
 
-There are a number of methods defined for _sync_ variables. Suppose _x_ is a
-sync variable of a given type,
+Discuss your observations.
```

Comment:
"Observations" implies this should be a Challenge, not a Discussion---which is generally a thought exercise.

Reply:
Ok, already edited this part out in my response in the previous conversation. Resolved through a commit.